README.md

```diff
@@ -65,17 +65,13 @@ information in the console at
 Manage > Access (IAM) > Access groups > Access policies.
 -->
 
-<!--
 You need the following permissions to run this module.
 
 - Account Management
```

Comment:
remove Account Management here since it only requires IAM Services permissions and not account level ones